lib/Parser.js

```diff
@@ -634,14 +634,14 @@ class Parser extends Tapable {
 		statement.params.forEach(param => {
 			this.walkPattern(param);
 		});
-		this.inScope(statement.params, function() {
+		this.inScope(statement.params, () => {
```

Comment:
Looks like the function is already anonymous and doesn't mess with preexisting functionality.  Although we could separated out the function and add in a name for debugging purposes.